src/Microsoft.Management.Deployment/PackageCatalogReference.cpp

```diff
@@ -15,42 +15,42 @@
 #include <winget/GroupPolicy.h>
 #include <AppInstallerErrors.h>
 #include <AppInstallerStrings.h>
+#include <winget/UserSettings.h>
 #include <Helpers.h>
 
 namespace winrt::Microsoft::Management::Deployment::implementation
 {
-    namespace
+    void PackageCatalogReference::Initialize(winrt::Microsoft::Management::Deployment::PackageCatalogInfo packageCatalogInfo, ::AppInstaller::Repository::Source sourceReference)
     {
-        std::string GetCallerName()
-        {
-            // See if caller name is set by caller
-            static auto callerName = GetComCallerName("");
+        static constexpr winrt::Windows::Foundation::TimeSpan s_PackageCatalogUpdateIntervalDelay = 168h; //1 week

```

Comment:
We might want to randomize this a bit as well, at least + 0/24 hours (worth of seconds or smaller) but maybe even up to a week extra.  That will hopefully prevent this from causing time based scheduled tasks to align on updates.